project-base/storefront/components/Blocks/Skeleton/SkeletonPageStore.tsx

```diff
@@ -1,7 +1,7 @@
 import { Webline } from 'components/Layout/Webline/Webline';
 import Skeleton from 'react-loading-skeleton';
 
-export const StorePageSkeleton: FC = () => (
+export const SkeletonPageStore: FC = () => (
```

Comment:
My idea behind this naming is not about proper english, rather it's about better visibility for what is page skeleton to not mix it. Other solution can be split then into separate folders. But in case we don't want folders I wouldn't care about proper english ordering since is help with grouping types of skeletons. We can discuss this with other guys.

Reply:
Yeah, okay. TBH in this I do not care that much, so if you have a reason behind it, that's okay.